osu.Game/Graphics/Backgrounds/SeasonalBackgroundLoader.cs

```diff
@@ -41,7 +41,11 @@ private void load(OsuConfigManager config, SessionStatics sessionStatics)
             seasonalBackgroundMode.BindValueChanged(_ => SeasonalBackgroundChanged?.Invoke());
 
             seasonalBackgrounds = sessionStatics.GetBindable<APISeasonalBackgrounds>(Static.SeasonalBackgrounds);
-            seasonalBackgrounds.BindValueChanged(_ => SeasonalBackgroundChanged?.Invoke());
+            seasonalBackgrounds.BindValueChanged(response =>
+            {
+                if (response.NewValue?.Backgrounds?.Count > 0)
```

Comment:
Out of curiosity, when does this become an issue? Looking at the usages of the bindable, it is only updated when an API request returns successful response, making this conditional impossible to be false?

Reply:
It happens 100% on dev server which returns zero backgrounds.

You make a good point though - it is likely still triggering an unnecessary change if a user has seasonal off and any arrive.